cmd/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import cmd "github.com/paulscherrerinstitute/scicat/cmd/commands"
+
+func main() {
```

Comment:
could we test here that when we call this main with an arg (e.g. datasetIngestor) the datasetIngestor from `commands` is executed?

Reply:
I have done something similar to that in the latest commit (calling Execute() instead of main, but it does effectively the same thing)